src/ui/datalens/pages/UserProfilePage/UserProfilePage.tsx

```diff
@@ -0,0 +1,80 @@
+import {unstable_Breadcrumbs as Breadcrumbs} from '@gravity-ui/uikit/unstable';
+// import {I18n} from 'i18n';
+import React from 'react';
+import block from 'bem-cn-lite';
+import {ActionPanel, DL, PageTitle} from 'ui';
```

Comment:
It's better do not use import from 'ui' because of increasing future problems with imports loop